Makefile

```diff
@@ -12,12 +12,23 @@ SCOPE_EXPORT=scope.tar
 SCOPE_UI_BUILD_EXPORT=scope_ui_build.tar
 SCOPE_UI_BUILD_IMAGE=$(DOCKERHUB_USER)/scope-ui-build
 SCOPE_VERSION=$(shell git rev-parse --short HEAD)
+DOCKER_VERSION=1.3.1
+DOCKER_DISTRIB=docker/docker-$(DOCKER_VERSION).tgz
+DOCKER_DISTRIB_URL=https://get.docker.com/builds/Linux/x86_64/docker-$(DOCKER_VERSION).tgz
 
 all: $(SCOPE_EXPORT)
 
-$(SCOPE_EXPORT): $(APP_EXE) $(PROBE_EXE) docker/*
+$(DOCKER_DISTRIB):
+	curl -o $(DOCKER_DISTRIB) $(DOCKER_DISTRIB_URL)
+
+docker/weave:
```

Comment:
This is just downloading the latest stable weave script to put it in the scope container; if the weave integration is enabled (automatically detected by the scope script), this script is used to get the weave ip addresses of the containers.